README.md

```diff
@@ -3,22 +3,24 @@
 [![Travis](https://travis-ci.org/JuliaDiff/ChainRules.jl.svg?branch=master)](https://travis-ci.org/JuliaDiff/ChainRules.jl)
 [![Coveralls](https://coveralls.io/repos/github/JuliaDiff/ChainRules.jl/badge.svg?branch=master)](https://coveralls.io/github/JuliaDiff/ChainRules.jl?branch=master)
 
-**Docs:* 
+**Docs:**
 [![](https://img.shields.io/badge/docs-master-blue.svg)](https://JuliaDiff.github.io/ChainRules.jl/dev)
 [![](https://img.shields.io/badge/docs-stable-blue.svg)](https://JuliaDiff.github.io/ChainRules.jl/stable)
 
 The ChainRules package provides a variety of common utilities that can be used by downstream automatic differentiation (AD) tools to define and execute forward-, reverse-, and mixed-mode primitives.
 
-This package is a WIP; the framework is essentially there, but there are a bunch of TODOs, virtually no tests, etc. PRs welcome! Documentation is incoming, which should help if you'd like to contribute.
+The Core logic of ChainRules is implemented in [ChainRulesCore.jl](https://github.com/JuliaDiff/ChainRulesCore.jl).
+If adding ChainRules support to your package (defining `rrule`s or `ffules` you only need to depend on that very light-weight package.
+This repo: ChainRules.jl is what people actually use, it reexports all the ChainRules functionality,
```

Comment:
can we make these full sentences? e.g.
```suggestion
This repository contains ChainRules.jl, which is what people actually use directly. 
ChainRules reexports all the ChainRulesCore functionality, and has all the rules for the Julia standard library.
```
and one sentence per line makes diffs in follow-up PRs much nicer

Reply:
here @oxinabox